### PR TITLE
new snacks - Jelly Beans

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR
Adds packet of Jelly Beans - small bean-shaped sugar candies that takes it roots from USA. Now everyone on the station can taste them in the Discount Dan`s.

**Media**

зображення

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Tunguso4ka
- add: Check your local Discount Dan`s for new Jelly Beans.

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
Glad to see this came back in a form more immediately recognizable in most places of the world. As a totally-not-biased jelly bean enjoyer, it was a bummer when the first one was closed.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml [hunk 1]
```diff
@@ -370,6 +370,21 @@
...
 
+- type: entity
+  id: FoodSnackJellyBeans
+  parent: FoodSnackBase
+  name: jelly beans
+  description: Packet full of rainbow colored jelly beans.
+  components:
+  - type: FlavorProfile
+    flavors:
+    - bee
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml:381]: Why do they taste like bees?

Comment (brainfood1183) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml:381]: how do you know jellybeans dont taste like bees?

Comment (Tunguso4ka) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml]: (unbeelivable sounded cool to me) fixed.

**Comment by metalgearsloth** [comment]
This still looks like https://www.roshen.com/ro/en/products/boxed-sweets/crazy-bee-frutty and isn't recognisable as jelly beans considering their distinct shape and bright colors https://www.google.com/search?q=jelly+beans&client=firefox-b-d&sxsrf=APwXEdc8kX3647HUS-HNhYu5cTNtMj9-QA:1682865185387&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiD9NCs6dH-AhWWpVYBHW_fATsQ_AUoAnoECAEQBA&biw=1411&bih=1456&dpr=1.36#imgrc=uT9Rxny-9YNgxM

**Comment by metalgearsloth** [comment]
@Tunguso4ka are you coming back to this?

**Comment by Tunguso4ka** [comment]
> @Tunguso4ka are you coming back to this?

i dont think so... You can close it.

**Pull request closed without merging**